docs/logic/mapper.md

```
@@ -72,6 +82,26 @@ extension UserDtoListMapper on List<UserDto>? {
 
+### 📌 Map → DTO 변환
+
+```dart
+extension MapToUserDto on Map<String, dynamic> {
+  UserDto toDto() => UserDto.fromJson(this);
+}
+```
+
+---
+
+### 📌 List<Map<String, dynamic>> → List<DTO> 변환
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: DTO

(MD033, no-inline-html)